.env.development

```diff
@@ -4,6 +4,14 @@ DS_AUTH_DATABASE_HOST=localhost
 DS_AUTH_DATABASE_PASSWORD=''
 DS_AUTH_DATABASE_USERNAME='postgres'
 
+ROTA_UID=8d142261e79fd579777c83bba6deffb9dea9499e696b41c839d101718819a48794d6b359bd51eae94b47c04fc1
+ROTA_SECRET=b436b09309d51f3d713381d0992b5f81a01b2bc1d778b6f8832d65fc949839c3c495a9bf5bf5bbe1aaa02f2522
+ROTA_REDIRECT_URI='http://rota.staging.defence-request.dsd.io'
```

Comment:
It could be, but in that case there is no standard at the moment and you could just override it in `.env.local`. I thought the staging urls would be the most sensible default.